lib/Core/Site/Values/ContentInfo.php

```diff
@@ -153,11 +153,15 @@ public function __debugInfo()
 
     public function getLocations($limit = 25)
     {
+        @trigger_error('getContent() is deprecated since version 2.6 and will be removed in 3.0. Use the same method on Content object instead.', E_USER_DEPRECATED);
```

Comment:
Wrong method name

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/87/commits/6434f02fc210e8e8450c1f8b8589b95bd33cd321